pages/api/resources/jobExplorer/fetchJobDetailsData.ts

```diff
@@ -0,0 +1,13 @@
+import { ANDROID_DEVELOPER_DATA } from "./androidDeveloper";
+import { IOS_DEVELOPER_DATA } from "./iosDeveloper";
+import { REACT_NATIVE_DEVELOPER_DATA } from "./reactNativeDeveloper";
+
+const jobDetailsMap = {
+  reactNativeDeveloper: REACT_NATIVE_DEVELOPER_DATA,
+  iosDeveloper: IOS_DEVELOPER_DATA,
+  androidDeveloper: ANDROID_DEVELOPER_DATA,
```

Comment:
as you remove the old page components, and start moving them all to use the reusable components, you will need to add to this map